code/src/styles/MovieStyle.css

```diff
@@ -0,0 +1,359 @@
+	align-self: end;
+
+	padding: 10px;
+}
+
+.details__rate {
+	color: #db0000;
+	font-size: 16px;
+	margin-left: 10px;
+}
+
+.details__tagline {
+	font-style: italic;
+}
+
+.details__imdb {
+	height: 45px;
+}
+
+.details__tag-lists {
+	display: flex;
+	flex-wrap: wrap;
+}
+
+.details__tag-list-color {
+	list-style: none;
+	padding: 3px;
+	margin: 5px 5px 10px 0;
+	background: rgb(75, 70, 70);
+}
+
+.company {
+	margin-top: 20px;
+	max-width: 1000px;
+}
+
+.company-container {
+	display: flex;
+	flex-wrap: wrap;
+}
+.company__names {
+	padding: 3px;
+	margin: 5px 5px 10px 0;
+	background: #1f6f8b;
+}
+
+/***BACKBUTTON ***/
+.back__button {
+	border-radius: 4px;
+	background-color: rgba(0, 0, 0, 0.5);
+	position: absolute;
+	border: none;
+	color: #fff;
+	text-align: center;
+	font-size: 20px;
+	padding: 10px 15px;
+	transition: all 0.5s;
+	cursor: pointer;
+	margin: 20px;
+	box-shadow: 0 10px 20px -8px rgba(0, 0, 0, 0.7);
+	cursor: pointer;
+	display: inline-block;
+	transition: 0.5s ease-in-out;
+	z-index: 1;
+	font-family: 'Nanum Gothic Coding', monospace;
+}
+
+.back__button:after {
+	content: '\00AB	';
+	position: absolute;
+	opacity: 0;
+	top: 10px;
+	left: -20px;
+	transition: 0.5s;
+}
+
+.back__button:hover {
+	padding-left: 25px;
+}
+
+.back__button:hover:after {
+	opacity: 1;
+	left: 5px;
+}
+
+/*** LOADER ***/
+.loader {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	height: 100vh;
+}
+
+.loader__icon {
+	border: 5px solid #f2f0f0;
+	border-radius: 50%;
+	border-top: 6px solid #1f6f8b;
+	width: 120px;
+	height: 120px;
+	top: 30px;
+	animation: spin 2s linear infinite;
+}
+
+/*** ERROR ***/
+.error {
+	background-image: linear-gradient(rgba(0, 0, 0, 0) 10%, rgb(0, 0, 0) 85%),
+		url('./popcorn.jpg');
+	background-size: cover;
+	background-position-x: center;
+	background-position-y: 45%;
+	height: 100vh;
+	width: 100vw;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+}
+
+.error__text {
+	font-size: 20px;
+	margin-bottom: 10px;
+	text-align: center;
+}
+
+.error__button {
+	background: rgb(216 61 61 / 60%);
+	padding: 20px;
+	font-size: 20px;
+	font-weight: bold;
+	border-radius: 10px;
+}
+
+/***ANIMATION***/
+/*Loader animation*/
+@keyframes spin {
+	0% {
+		transform: rotate(0deg);
+	}
+	100% {
+		transform: rotate(360deg);
+	}
+}
+
+/*** MEDIA ***/
+@media (min-width: 768px) {
+	h1 {
+		font-size: 32px;
+	}
+
+	h3 {
+		font-size: 26px;
+	}
+
+	h4 {
+		font-size: 20px;
+	}
+
+	p {
+		font-size: 18px;
+	}
+	.nav__button {
+		display: none;
+	}
+
+	.nav__link {
+		font-size: 20px;
+	}
+	.nav__button--open {
+		all: unset;
+		display: flex;
+	}
+
+	.nav__button--closed {
+		display: flex;
+	}
+
+	.list {
+		grid-template-columns: repeat(3, 1fr);
+	}
+
+	.details__img {
+		display: flex;
+		border: 1px solid #f8f3ea;
+	}
+
+	.details__info {
+		grid-column: 2/4;
+	}
+}
+
+@media (min-width: 1024px) {
+	.list {
+		grid-template-columns: repeat(4, 1fr);
+	}
+}
```

Comment:
I noticed that you have four posters in a row at most on bigger screens. However because the intrinsic size of the poster image is 300*425px, the resolution of the posters gets blurry on screens larger than 1200px vw because the image width then gets up to 400px+. A solution would be to add 5 grid-template-columns for screens above 1200px.